docs/docs/gatsby-image.md

```diff
@@ -0,0 +1,351 @@
+- `cropFocus` (string, default: `[sharp.strategy.attention][6]`)
+- `pngCompressionSpeed` (int, default: 4)
+
+Here's an example of using the `duotone` option with a fixed image:
+
+```graphql
+fixed(
+  width: 800,
+  duotone: {
+    highlight: "#f00e2e",
+    shadow: "#192550"
+  }
+)
+```
+
+Read more in the [`gatsby-plugin-sharp`](/packages/gatsby-plugin-sharp) README.
+
+## Image query fragments
+
+GraphQL includes a concept called "query fragments", which are a part of a query that can be reused. To ease building with `gatsby-image`, Gatsby image processing plugins which support `gatsby-image` ship with fragments which you can easily include in your queries.
+
+> Note: using fragments in your queries depends on which data source(s) you have configured. Read more in the [gatsby-image](/packages/gatsby-image#fragments) README.
+
+### Common fragments with `gatsby-transformer-sharp`
+
+#### Fixed images
+
+- `GatsbyImageSharpFixed`
+- `GatsbyImageSharpFixed_noBase64`
+- `GatsbyImageSharpFixed_tracedSVG`
+- `GatsbyImageSharpFixed_withWebp`
+- `GatsbyImageSharpFixed_withWebp_noBase64`
+- `GatsbyImageSharpFixed_withWebp_tracedSVG`
+
+#### Fluid images
+
+- `GatsbyImageSharpFluid`
+- `GatsbyImageSharpFluid_noBase64`
+- `GatsbyImageSharpFluid_tracedSVG`
+- `GatsbyImageSharpFluid_withWebp`
+- `GatsbyImageSharpFluid_withWebp_noBase64`
+- `GatsbyImageSharpFluid_withWebp_tracedSVG`
+
+#### About `noBase64`
+
+If you don't want to use the [blur-up effect](https://using-gatsby-image.gatsbyjs.org/blur-up/), choose the fragment with `noBase64` at the end.
+
+#### About `tracedSVG`
+
+If you want to use the [traced placeholder SVGs](https://using-gatsby-image.gatsbyjs.org/traced-svg/), choose the fragment with `tracedSVG` at the end.
+
+#### About `withWebP`
+
+If you want to automatically use [WebP images](https://developers.google.com/speed/webp/) when the browser supports the file format, use the `withWebp` fragments. If the browser doesn't support WebP, `gatsby-image` will fall back to the default image format.
+
+Here's an example of using a non-default fragment from `gatsby-transformer-sharp`. Be sure to pick one that matches your desired image type (_fixed_ or _fluid_):
+
+```graphql
+file(relativePath: { eq: "images/default.jpg" }) {
+  childImageSharp {
+    fluid {
+      // highlight-next-line
+      ...GatsbyImageSharpFluid_tracedSVG
+    }
+  }
+}
+```
+
+For more info on how these options work, check out the Gatsby Image demo: [https://using-gatsby-image.gatsbyjs.org/](https://using-gatsby-image.gatsbyjs.org/)
+
+#### Additional plugin fragments
+
+Additionally, plugins supporting `gatsby-image` currently include [gatsby-source-contentful](/packages/gatsby-source-contentful/), [gatsby-source-datocms](https://github.com/datocms/gatsby-source-datocms) and [gatsby-source-sanity](https://github.com/sanity-io/gatsby-source-sanity). See the [gatsby-image](/packages/gatsby-image/#fragments) README for more details.
+
+## Gatsby-image props
```

Comment:
It looks like this info (and maybe some other stuff in this API section are really closely related to what's in the `gatsby-image` README which this page links to, would it make sense to have the README link back to this page too? 

And maybe this table of props is something that could be an MDX component at some point so the two don't diverge (though I'm guessing they don't get updated frequently if at all).